lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -60,6 +61,28 @@ function getNodesAndTimingByUrl(nodeTimings) {
   return urlMap;
 }
 
+/**
+ * @param {Node} node
+ * @param {LH.Gatherer.Simulation.NodeTiming} nodeTiming
+ * @param {LH.Artifacts.DetectedStack[]} Stacks
+ */
+function computeStackSpecificTiming(node, nodeTiming, Stacks) {
+  const stackSpecificTiming = Object.assign({}, nodeTiming);
+  if (!Stacks) return stackSpecificTiming;
+  if (Stacks.some(stack => stack.id === 'amp')) {
+    // AMP will load a linked stylesheet asynchronously if it has not been loaded after 1 second [https://github.com/ampproject/amphtml/blob/master/src/font-stylesheet-timeout.js]
```

Comment:
based on their latest master it actually looks like they cap the stylesheet end load time to 2.1s now instead of the stylesheet load time to 1s which is pretty neat, we'll need to adjust our logic accordingly